src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -449,6 +433,36 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		}
 	}
 
+	private loadContentMetadata(metadata: any): void {
```

Comment:
This should be INotebookMetadata instead of any, correct? We should avoid using any whenever possible

Reply:
The metadata field was originally type any because azdata_notebook_guid is not a field in INotebookMetadata. https://github.com/microsoft/azuredatastudio/blob/8f5dc1526aaf1f649978b95ce19607449c6465c6/src/sql/workbench/services/notebook/browser/models/notebookModel.ts#L386
@corivera would it be ok to add azdata_notebook_guid to INotebookMetadata and expectedKeys?

Reply:
We probably shouldn't add that to INotebookMetadata since that's really only for internal use.

I'd suggest making an internal metadata type that extends INotebookMetadata with the private internal properties like that. Then just have the implementations return that type instead where it loads the content for a notebook.